typescript/interchain/.gitignore

```diff
@@ -0,0 +1,9 @@
+artifacts/
```

Comment:
Consider renaming the directory to `ica` or something more specific. "interchain" is ambiguous as it covers a lot of things that we do :)